docs/elastic_search.md

```diff
@@ -303,3 +469,165 @@ however in practice plural words tend to be covered by the singular replacement
 (e.g. "add on" is a substring in "add ons",
 so "add ons" is replaced by "addons")
 and there is marginal benefit to defining _every single_ conjugation of a verb.
+
+## Production Deployment
```

Comment:
I would remove this completely. This is one possible way to do a major upgrade and is related to the current setup that we have. There are many others. It's not related to how search work and it should not dictate the policy on how to perform major version upgrades